contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -56,6 +56,14 @@ LANGUAGE plpgsql;
  * final behaviour.
  */
 
+-- 3.1 introduces a change to how table types are stored in pg_type. While MVU forces the catalog
+-- to be regenerated, mVU (minor version upgrade) does not, so we need to manually fix it here.
+UPDATE pg_type SET typbyval = 't' 
+FROM sys.babelfish_namespace_ext AS ns 
+WHERE typtype = 'c' AND
+    typacl IS NOT NULL AND
+    typacl::TEXT LIKE '%' || ns.nspname || '%';
```

Comment:
Using `like '%nspname%'` is not 100% safe because non-babelfish namespace could contain some babelfish namespace strings. Let's use an exact match using `babelfish_namespace_ext.nspname`, `pg_namespace.nspname`, `pg_namespace.oid` and `pg_type.typnamespace`.